### DISCUSSION TITLE
Disable password policy for single user in the realm

### DISCUSSION THREAD

**Discussion by alokrbl** [keycloak/keycloak#29049] [Q&A] [answered]
### Description

I have a requirement, where i need to have a user with no password expiry and all realm level password policy should remain intact for other users.
I am unable to achieve this in any way. Is there any existing way by which i can achieve this or this feature is not supported by keycloak? Do we have any plan to have this feature?

Please assist.

### Motivation

I want to have one non-human user for admin/automation purpose without password expiry.

### Details

I am not sure if we have admin REST API support to have custom required_action for overriding password policy for one user to override the default password policy in the realm.

**Comment by codespearhead** [comment, marked as answer]
Machine-to-machine communication should be carried out via [Service Accounts](https://www.keycloak.org/docs/latest/server_admin/#_service_accounts).

**Reply by alokrbl** [reply]
Thanks @codespearhead
